201-vmss-windows-customimage/azuredeploy.json

```diff
@@ -0,0 +1,290 @@
+				"Standard_DS13",
+				"Standard_D14",
+				"Standard_DS14"
+			],
+			"metadata": {
+				"description": "The size of the VM instances Created"
+			}
+		},
+		"dnsNamePrefix": {
+			"type": "string",
+			"metadata": {
+				"description": "The Prefix for the DNS name of the new IP Address created"
+			}
+		},
+		"adminUsername": {
+			"type": "string",
+			"metadata": {
+				"description": "The Username of the admininstrative user for each VM instance created"
+			}
+		},
+		"adminPassword": {
+			"type": "securestring",
+			"metadata": {
+				"description": "The Password of the admininstrative user for each VM instance created"
+			}
+		},
+		"location": {
+			"type": "string",
+			"metadata": {
+				"description": "Deployment location"
+			}
+		},
+		"imageSource": {
+			"type": "string",
+			"metadata": {
+				"description": "The source of the blob containing the custom image"
+			}
+		},
+		"frontEndLBPort": {
+			"type": "int",
+			"metadata": {
+				"description": "The front end port to load balance"
+			},
+			"defaultValue": 80
+		},
+		"backEndLBPort": {
+			"type": "int",
+			"metadata": {
+				"description": "The front end port to load balance"
+			},
+			"defaultValue": 80
+		},
+		"probeIntervalInSeconds": {
+			"type": "int",
+			"metadata": {
+				"description": "The interval between load balancer health probes"
+			},
+			"defaultValue": 15
+		},
+		"numberOfProbes": {
+			"type": "int",
+			"metadata": {
+				"description": "The number of probes that need to fail before a VM instance is deemed unhealthy"
+			},
+			"defaultValue": 5
+		},
+		"probeRequestPath": {
+			"type": "string",
+			"metadata": {
+				"description": "The path used for the load balancer health probe"
+			},
+			"defaultValue": "/iisstart.htm"
+		}
+	},
+	"variables": {
+		"addressPrefix": "10.0.0.0/16",
+		"subnetName": "Subnet",
+		"subnetPrefix": "10.0.0.0/24",
+		"virtualNetworkName": "vmssvnet",
+		"vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
+		"subnetRef": "[concat(variables('vnetID'),'/subnets/',variables('subnetName'))]",
+		"publicIPAddressName": "publicip1",
+		"publicIPAddressID": "[resourceId('Microsoft.Network/publicIPAddresses',variables('publicIPAddressName'))]",
+		"nicName": "networkInterface1",
+		"nicId": "[resourceId('Microsoft.Network/networkInterfaces',variables('nicName'))]",
+		"lbName": "loadBalancer1",
+		"lbID": "[resourceId('Microsoft.Network/loadBalancers',variables('lbName'))]",
+		"lbFEName": "loadBalancerFrontEnd",
+		"lbWebProbeName": "loadBalancerWebProbe",
+		"lbBEAddressPool": "loadBalancerBEAddressPool",
+		"lbFEIPConfigID": "[concat(variables('lbID'),'/frontendIPConfigurations/',variables('lbFEName'))]",
+		"lbBEAddressPoolID": "[concat(variables('lbID'),'/backendAddressPools/',variables('lbBEAddressPool'))]",
+		"lbWebProbeID": "[concat(variables('lbID'),'/probes/',variables('lbWebProbeName'))]"
+	},
+	"resources": [
+		{
+			"apiVersion": "2015-06-15",
```

Comment:
It's a good idea to make the apiVersion a variable for easier migration to newer versions